app/views/user/camps_registrations/medical_information.html.erb

```diff
@@ -0,0 +1,49 @@
+<%= render 'progress' %>
+<h1>MEDICAL INFORMATION</h1>
+<%= form_for @camps_registration, url: wizard_path, method: :put do |f| %>
+
+  <%= f.label "Do you have any disability ?" %>
+  <div class="row">
+    <div class="col-sm-2" id="disability-yes">
+      <%= f.radio_button("disability", "yes", checked: (@camps_registration.disability != "")) %>
```

Comment:
Can we replace `(@camps_registration.disability != ""))` with `@camps_registration.present?`